src/site.webmanifest

```diff
@@ -0,0 +1,8 @@
+{
+    "icons": [{
+        "src": "icon.png",
+        "sizes": "192x192",
+        "type": "image/png"
+    }],
+    "start_url": "/"
```

Comment:
I think it's a bit risky to default to "/", unless you are really sure that most people will use HBP at the root of a domain. I would caution about using this line. 

For example, http://www.w3.org/Mobile/IG/ (app somewhere within a larger domain)